### PR TITLE
Corrected indentation of app/renderer/index.html

### PR DESCRIPTION
**Pull request by smddzcy**
## Changes:
- Added a missing html tag
- Corrected indentation

Issue: https://github.com/vutran/dext/issues/89

### CODE REVIEW

**Comment by adnasa** [comment]
> Added a missing html tag

Thanks for the contribution.. which is the "missing" html-tag? can't see it in the diff

File: app/renderer/index.html [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-<head>
-<title>Dext</title>
-<style type="text/css">
-html, body {
-  margin: 0px;
-  padding: 0px;
-}
-</style>
-</head>
+<html>
```
Comment (smddzcy) [inline app/renderer/index.html]: Here 👍

**Comment by adnasa** [comment]
aaaah, I see it now. thanks :)

**Comment by adnasa** [comment]
merging..

**Comment by smddzcy** [comment]
it was tough 😅 thanks 👍

**Pull request merged**